src/d1to2fix/symbolsearch.d

```diff
@@ -1,27 +1,38 @@
+/******************************************************************************
+
+    Module built on top of dsymbol library facilities which provides means
+    to find D symbols and reasons about them.
+
+    Copyright: Copyright (c) 2016 Sociomantic Labs. All rights reserved
+
+    License: Boost Software License Version 1.0 (see LICENSE for details)
+
+******************************************************************************/
+
 module d1to2fix.symbolsearch;
 
+/******************************************************************************
+
+    Imports
+
+******************************************************************************/
+
```

Comment:
Pfff, I would like to eventually get rid of this imports header, is utterly useless.